imagemounter/dependencies.py

```diff
@@ -0,0 +1,153 @@
+from imagemounter import _util
+
+
+class Dependency(object):
+    installed_explanation = "INSTALLED {!s}"
```

Comment:
There is some discrepancy between the ``installed_explanation`` (which is a string that needs to be formatted and includes the INSTALLED part) and ``missing_explanation`` (which is a formatted string that does not include the MISSING part). That feels a bit odd.

I think it is better to have a single fully formatted ``explanation``, without the INSTALLED/MISSING part and without the module name, and include that in the ``status`` property.

Reply:
👍 . I was trying to be too tricky with code reuse and subclass inheritance. I agree this would make the code more understandable.